compiler/lsp-tests/src/Main.hs

```diff
@@ -401,6 +404,19 @@ stressTests run _runScenarios = testGroup "Stress tests"
                 ]
             expect :: Int -> Session ()
             expect i = when (odd i) $ do
+                -- Despite these checks succeeding, the rest of the
+                -- block can fail for failing to read 'hlint.yaml'.
+                -- It's read at the end of the day by
+                -- Data.Yaml.decodeFileEither. Is there some sort of
+                -- contention going on?
+                hlintDataDir <- liftIO getHlintDataDir
```

Comment:
`getHlintDataDir` depends on the executable you run it from so this test isn’t testing what you think it is. It tests if the test suite executable finds the files not if damlc finds the files.

Reply:
This is debugging code that I used to assure myself that the configuration file exists where it's expected to be. I put it there because reading the config file is failing in the stress test and wanted to assure myself that it wasn't for lack of its existence. I'm very confused why these tests are failing

Reply:
Any way, time to take this debugging code out.